src/seedu/addressbook/data/AddressBook.java

```diff
@@ -61,6 +61,11 @@ public void clear() {
         allPersons.clear();
     }
 
+    /**
+     * Sorts all persons in address book.
+     */
+    public void sort() {allPersons.sort(); }
```

Comment:
You can start a new line after the opening curly brace, and closed curly brace should occupy a line itself.